src/main/java/Duke.java

```diff
@@ -1,10 +1,59 @@
+import java.awt.datatransfer.StringSelection;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.List;
+
 public class Duke {
+    static void printHello() {
+        System.out.println("\tSup! I'm Air\n" + "\tHow can I help you out today?\n");
+        System.out.println("____________________________________________________________\n");
```

Comment:
You can consider refactoring your strings as constant instead, such as MESSAGE_GREETING, would make the code more readable and better for UI/UX design in the future.